ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!--
+************ STOP!! ************
+If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
+the Apache Software Foundation Security Team at security@trafficcontrol.apache.org and follow the
+guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+-->
+
+<!--
+- For *SUPPORT QUESTIONS*, use the
+[Traffic Control slack channels](https://traffic-control-cdn.slack.com) or [Traffic Control mailing lists](http://trafficcontrol.apache.org/mailing_lists/).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR. -->
+
+**I'm submitting a ...**
+<!-- (check one with "x") -->
+<!--- security vulnerability (STOP!! - see above)-->
+- [ ] bug report
```

Comment:
personally I think the options ought to get more specific as you go down. Consider this scenario: a user notices that Profile Import no longer works in TPv4. They open an issue, and read through the template. "I'm not reporting a security vulnerability... this isn't a support question... check one type of issue I'm opening, okay, got it... ah! bug report, that was easy" and then they move on to the next section.

Reply:
open to suggestions. how you would list them?

Reply:
i changed them slightly

Reply:
that looks fine.